images/create.md

```diff
@@ -0,0 +1 @@
+a
```

Comment:
can it be removed?

Reply:
Yeah, sorry, I thought I deleted that. It's deleted now.

It is necessary to create this file when using the GitHub Web UI because it's not possible to create a new directory when uploading an image, so to create a new directory for images you have to create a text file and the new directory at the same time, then upload the image(s), then delete the text file. Crazy, I know, but that's how it is.